runtime/src/main/java/io/quarkiverse/cxf/CXFClientData.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkiverse.cxf;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.jboss.logging.Logger;
+import org.wildfly.common.annotation.Nullable;
+
+/**
+ * Provides runtime metadata for a CXF client.
+ *
+ * <p>
+ * This class contains extracted from a SEI. It contains basic data to
+ * setup a proxy client for a given SEI.
+ * </p>
+ *
+ * @author wh81752
```

Comment:
Kindly remove all author tags in various classes.  There is awesome view of all contributors to a project at the following link:  https://github.com/quarkiverse/quarkus-cxf/graphs/contributors

Reply:
Done.

Reply:
Done.